siege_utilities/geo/spatial_transformations.py

```
@@ -219,24 +228,33 @@ def _convert_to_duckdb(self, gdf: GeoDataFrame, **kwargs) -> bool:
             return False
             
         try:
-            # Get database parameters from user config or kwargs
-            db_path = kwargs.get('db_path') or self.user_config.get_database_connection('duckdb') or ':memory:'
+            # Get database parameters from user config or kwargs.
+            db_path = kwargs.get('db_path')
+            if db_path is None and self.user_config is not None:
+                try:
+                    db_path = self.user_config.get_database_connection('duckdb')
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)